src/commands.ts

```diff
@@ -20,8 +20,6 @@ export namespace Commands {
 
     export const VIEW_PACKAGE_REFRESH = "java.view.package.refresh";
 
-    export const VIEW_PACKAGE_OPEN_FILE = "java.view.package.openFile";
-
     export const VIEW_PACKAGE_OUTLINE = "java.view.package.outline";
```

Comment:
Just curious, we have so many constants in this file, is there anyone that is not referenced but still exists in this file?

Reply:
Just did a check, all command constants are used at least once.